package-scripts.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
     dev: {
       script: series([
         startInNewWindow('npm start dev.mongo --silent'),
-        startInNewWindow('cross-env PORT=8080 npm start dev.client --silent'),
+        startInNewWindow('./node_modules/.bin/cross-env PORT=8080 npm start dev.client --silent'),
```

Comment:
ah, looks like this is too long. Try running `npm start format`

Also, you may wanna update your contributions in `.all-contributorsrc` to include `code` :)